### PR TITLE
Logs

### PR DESCRIPTION
**Pull request by IGLADI**
Logging system & auto repair received XMLs (also our code doesn't care anymore about xmlns=...)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,10 +24,10 @@ Before running the script, make sure you have a RabbitMQ instance running. If th
 docker compose -f ./extra/rabbitmq_general_team/docker-compose.yml up -d
 ```
 
-Before running any of the next commands, please make sure you're in the `src` folder with the following command:
+Before running any of the next commands, please make sure you're in the `/monitoring/Monitoring/src` folder (or similar if you have placed the project somewhere else) with the following command:
 
 ```bash
-cd src
+cd /monitoring/Monitoring/src
```
Comment (luluwaza) [inline README.md:30]: good looking guy that wrote this

File: extra/rabbit_mq_hello_world/publisher.py [hunk 2]
```diff
@@ -8,7 +8,13 @@ def send_message(message):
         password = env.readline().strip()
     credentials = pika.PlainCredentials("mrgydtsi", password)
 
-    connection = pika.BlockingConnection(pika.ConnectionParameters(host="rat.rmq2.cloudamqp.com", virtual_host="mrgydtsi", credentials=credentials))
+    connection = pika.BlockingConnection(
```
Comment (luluwaza) [inline extra/rabbit_mq_hello_world/publisher.py:11]: Maybe add some explanation here

**Review by luluwaza — APPROVED** [review]
Great changes, more readable code, very nice!

File: extra/rabbit_mq_hello_world/publisher.py (hunk 2 above)
Comment (IGLADI) [reply extra/rabbit_mq_hello_world/publisher.py:11]: Was just the hello world to learn & test, isn't running on the real server.

File: README.md (hunk 1 above)
Comment (IGLADI) [reply README.md:30]: Think that's you bro🤨

**Pull request merged**